docs/errorHandling.md

```
@@ -0,0 +1,37 @@
+# Error Handling
+
+# Error Handling
```

markdownlint-cli2 (0.23.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)